src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2545,6 +2545,8 @@ void ApplicationManagerImpl::UnregisterApplication(
     MessageHelper::SendUnsubscribedWayPoints(*this);
   }
 
+  EndNaviServices(app_id);
```

Comment:
@Ypostolov What if application did not use AUDIO/VIDEO services? 
 (f.e. application can use RPC service ONLY)

Reply:
@AByzhynar 
In this case (application did not use AUDIO/VIDEO services) this function do nothing, just print error message in log : "No info about navi servicies for app".